onager/meta_launcher.py

```diff
@@ -133,5 +161,7 @@ def meta_launch(args):
             print(cmd)
         jobs[i] = (cmd,tag)
 
+    print(f"Prelaunched {len(jobs)} jobs for {args.jobname}.")
+
```

Comment:
Lol stop sneaking in other functionality! Does this work correctly with the `+quiet` argument?